css/style.css

```diff
@@ -1,7 +1,316 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+ }
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+html. body {
+	height: 100%;
+}
+
+body {
+		margin: 0 auto; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455736)